### PR TITLE
Added Setup and Deployment steps to the readme.

### PR DESCRIPTION
**Pull request by wahibkapdi**
Updated the Readme to include the setup and deployment steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,115 @@ The key stakeholder for this application is the Associate Department Head, who i
...
+
+---
+
+## 2. Install Dependencies and setup Rails Master Key
+Run `bundle install` to install all required dependencies.
+
+```bash
+bundle install
+```
+Generate your ```GOOGLE_CLIENT_ID``` and ```GOOGLE_CLIENT_SECRET```. Encrypt both of these using a ```RAILS_MASTER_KEY``` and store it in your ```config/credentials.ymc.enc```
```
Comment (colbyendres) [inline README.md]: This is done on the Google Developer console, not through Rails. This [link](https://github.com/tamu-edu-students/Google-Auth-Ruby-By-JD#setup-google-oauth-on-googles-end) provides details on how to generate them

File: README.md [hunk 2]
```diff
@@ -19,3 +19,115 @@ The key stakeholder for this application is the Associate Department Head, who i
...
+## 2. Install Dependencies and setup Rails Master Key
+Run `bundle install` to install all required dependencies.
+
+```bash
+bundle install
+```
+Generate your ```GOOGLE_CLIENT_ID``` and ```GOOGLE_CLIENT_SECRET```. Encrypt both of these using a ```RAILS_MASTER_KEY``` and store it in your ```config/credentials.ymc.enc``` 
+Make sure you ```config/master.key``` popuplated with your ```RAILS_MASTER_KEY```.
+
+or just store your Google OAuth credentials in your ```config/credentials.ymc.enc``` unencrypted but do not commit them.
```
Comment (colbyendres) [inline README.md]: Should be `config/credentials.yml.enc`, where the `enc` suffix means that the file is encrypted. This file is automatically generated by `rails credential:edit`, so the user shouldn't manually create this

**Comment by colbyendres** [comment]
Looks about right to me. The build instructions are going to change to incorporate building the dependencies for the solver. It should just be one extra step (running `rake glpk:install`), so that will be an easy change.

**Comment by Naviunni** [comment]
Could you also add all the links to our sprint plans to the readme? The sprint1 plan is currently linked
Also, please remove the sprint 1 goal

File: README.md (hunk 1 above)
Comment (wahibkapdi) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (wahibkapdi) [reply README.md]: Done

**Comment by wahibkapdi** [comment]
> Looks about right to me. The build instructions are going to change to incorporate building the dependencies for the solver. It should just be one extra step (running `rake glpk:install`), so that will be an easy change.

We can add that once we have integrated the linear solver into the application.

**Comment by wahibkapdi** [comment]
Updated the Readme to include the setup and deployment steps.

**Review by navyapriyanandi-tamu — APPROVED** [review]

**Pull request merged**